integration_tests/steps/two_sided_market.py

```diff
@@ -199,3 +200,63 @@ def energy_origin(context, producer, consumer):
                 assert trade.seller_origin == producer
 
     assert trade_count > 0
+
+
+@then('trades are matched only on the Grid market')
+def trades_matched_on_grid(context):
+    # Assert that all grid trades contain Offer objects, bid trades are not tracked
+    assert all(type(trade.offer) == Offer
```

Comment:
Im wondering if there would be any consequences of using `isinstance` here

Reply:
There shouldn't be any actually, I'll give it a try to replace them. Thanks